jupyter-js-widgets/src/widget_box.js

```diff
@@ -229,34 +237,36 @@ var FlexBoxView = BoxView.extend({ // TODO: Deprecated in 5.0 (entire view)
     },
 
     update_orientation: function() {
-        var orientation = this.model.get("orientation");
-        if (orientation == "vertical") {
-            this.$box.removeClass("hbox").addClass("vbox");
+        var orientation = this.model.get('orientation');
+        if (orientation == 'vertical') {
+            this.box.classList.remove('hbox');
+            this.box.classList.add(vbox);
         } else {
-            this.$box.removeClass("vbox").addClass("hbox");
+            this.box.classList.remove('vbox');
+            this.box.classList.add('hbox');
```

Comment:
ClassList.add only takes one argument with IE. Besides, it is not defined
at all for elements of the svg namespace.

The only placr we really have to carr about svg is the top DOMWidgetView
and it is dealt with correctly.
On Mar 18, 2016 10:34 AM, "Jonathan Frederic" <notifications@github.com>
wrote:

> In jupyter-js-widgets/src/widget_box.js
> <https://github.com/ipython/ipywidgets/pull/427#discussion_r56664590>:
>
> >          } else {
> > -            this.$box.removeClass("vbox").addClass("hbox");
> > +            this.box.classList.remove('vbox');
> > +            this.box.classList.add('hbox');
>
> note: there is a classList polyfill floating around the web from MSDN
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ipython/ipywidgets/pull/427/files/65a4c73427af92a709170001b6b2e52119b14e5d#r56664590>
>